crates/hc_deepkey_sdk/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "hc_deepkey_sdk"
+version = "0.6.0"
+authors = ["Matthew Brisebois <matthew.brisebois@holo.host>"]
+edition = "2018"
+license = "CAL-1.0"
```

Comment:
Apache-2.0

Reply:
Why not CAL?

Reply:
Is anything downstream of this already released as Apache? (or plan to be in the future?) If so, this needs to be Apache. Otherwise, this could be CAL, since it's dealing with cryptography.

Reply:
I thought out rule was to use Apache for library crates unless they're actually creating/storing cryptographic keys. This SDK isn't doing either, it's just a client library. We did a pass of all the licenses last year and only a couple of things are using the CAL. Holochain is, and the HDI/HDK because they expose crypto operations

Reply:
I totally hear that argument, and am fine with Apache. And also, these are literally the structs being used to store the data in our public key infrastructure. Perhaps it makes sense for the use of these to require users to honor the CAL?

Reply:
Although, it's far easier to switch from Apache to CAL later, rather than the other way around... So maybe we should start with Apache as the more future flexible option.

Reply:
I prefer the licensing to allow people to build something that doesn't comply with the CAL with our libraries. If people choose to build apps using our HDK/HDI and our end product, Holochain, then they have to respect the rights of end users as required by the CAL. 

The argument for this being part of our PKI makes a lot of sense, but equally, if somebody builds something other than DPKI with this types crate, I don't mind. I think DPKI itself should be licensed CAL because that's what actually does the work. Even so though, that's embedded inside Holochain which is CAL licensed.

I'm so glad I'm not a lawyer but I think this puzzle just about makes sense to me :D

Reply:
I agree. I vote for Apache here.